cmd/api/routers/routers.go

```diff
@@ -110,6 +118,34 @@ func (c *Controller) GetNamespacedCoreResources(context *gin.Context) {
 	context.JSON(http.StatusOK, NewList(resources))
 }
 
+// Livez returns current server configuration
```

Comment:
I would like to document WHY it returns just the configuration. "Isn't this supposed to check if the app is able to operate?". What about:

```suggestion
// Livez returns current server configuration as we don't have a clear deadlock indicator
```

Reply:
Change applied